profiles/base.yaml

```diff
@@ -114,7 +114,8 @@ empty_hunting_room_messages:
 - Anyone here?
 hunting_buddies:
 stop_hunting_if_bleeding: true
-
+pre_hunt_buff: false
```

Comment:
I think it needs to be left here. Sometimes you have a waggle set, but you no longer suffer from mandatory pre-buff because the critter can't kill you before you are capable of buffing. This is a personal preference thing, prebuffing always could be good too.